WordPressAuthenticatorTests/GoogleSignIn/CodeVerifierTests.swift

```diff
@@ -0,0 +1,46 @@
+@testable import WordPressAuthenticator
+import XCTest
+
+class CodeVerifierTests: XCTestCase {
+
+    func testCodeVerifierIsRandomString() {
+        XCTAssertNotEqual(
+            ProofKeyForCodeExchange.CodeVerifier().rawValue,
```

Comment:
nit – could we make this `equatable` so that `rawValue` doesn't need to be exposed?

Or is that too much window dressing?

Reply:
I don't think it's window dressing. 

See 07a3b2b6

The compiler does all the work and the resulting code is more compact. Win for the tests and win for future consumers of the production code.